### PR TITLE
Img

### PR DESCRIPTION
**Pull request by bcail**

### CODE REVIEW

File: loris/webapp.py [hunk 1]
```diff
@@ -664,47 +666,35 @@ def get_img(self, request, ident, region, size, rotation, quality, target_fmt, b
...
+                #Redirect if appropriate
                 if self.redirect_canonical_image_request:
                     if not image_request.is_canonical(info):
                         self.logger.debug('Attempting redirect to %s', image_request.canonical_request_path,)
                         r.headers['Location'] = image_request.canonical_request_path
                         r.status_code = 301
                         return r
 
-                # 5. Make an image
+                #Make an image
```
Comment (alexwlchan) [inline loris/webapp.py]: nit: Python convention is to put a space after the `#` for comments.

File: loris/webapp.py [hunk 2]
```diff
@@ -664,47 +666,35 @@ def get_img(self, request, ident, region, size, rotation, quality, target_fmt, b
                 return r
         else:
             try:
-                # 1. Get the info
-                info = self._get_info(ident, request, base_uri)[0]
```
Comment (alexwlchan) [inline loris/webapp.py:668]: Nice catch!

**Review by alexwlchan — APPROVED** [review]
Looks good, nice that we don’t have to get the `info` twice. One nit, otherwise happy to merge.

**Comment by bcail** [comment]
thanks @alexwlchan

**Pull request merged**